tests/mypy/outputs/plugin-success-strict.txt

```diff
@@ -1,3 +1,4 @@
 29: error: Unexpected keyword argument "z" for "Model"  [call-arg]
 64: error: Untyped fields disallowed  [pydantic-field]
-79: error: Argument "x" to "OverrideModel" has incompatible type "float"; expected "int"  [arg-type]
\ No newline at end of file
+79: error: Argument "x" to "OverrideModel" has incompatible type "float"; expected "int"  [arg-type]
+125: error: Untyped fields disallowed  [pydantic-field]
```

Comment:
In current test-case, I write it without explicit type, because it is the only way when mypy's semanal thinks that it is TypeAlias

In my IRL case, we use it with strict mypy and inheritance, so mypy's semanal also thinks it is a TypeAlias, like this:

```python
class Parent(BaseModel):
    __type_alias_attribute__: type

class TypeAliasAsAttribute(Parent):
    __type_alias_attribute__ = Union[str, bytes]
```